File: go/1007.go
package main

import "fmt"

func main() {
	var a, b, c, d int
	fmt.Scan(&a)
	fmt.Scan(&b)
	fmt.Scan(&c)
	fmt.Scan(&d)
	var result int = a*b - c*d
	fmt.Printf("DIFERENCA = %v\n", result)
}

File: C/src/1007.c
#include <stdio.h>

int main(){
    int a,b,c,d;
    scanf("%i\n%i\n%i\n%i",&a,&b,&c,&d);
    int diferenca = a*b-c*d;
    printf("DIFERENCA = %i\n",diferenca);
    return 0;
}